server/helpers/subscriberUtils.js

```diff
@@ -154,32 +154,36 @@ const findTelegramSubscription = async (address, chatId) => {
 }
 
 const getSubscriptorRole = async subscriptor => {
+  console.log(`[Subscribers-utils] - getSubscriptorRole()`)
   const { getProtocolService } = require('./services/protocolService')
   const { getDelegatorService } = require('./services/delegatorService')
   const protocolService = getProtocolService()
   const delegatorService = getDelegatorService()
-
-  let [constants, delegator] = await promiseRetry(retry => {
-    return Promise.all([
-      protocolService.getLivepeerDefaultConstants(),
-      delegatorService.getDelegatorAccount(subscriptor.address)
-    ]).catch(err => retry())
-  })
-
-  const { status, address, delegateAddress } = delegator
-
-  // Detect role
-  const role =
-    delegator &&
-    status === constants.DELEGATOR_STATUS.Bonded &&
-    delegateAddress &&
-    address.toLowerCase() === delegateAddress.toLowerCase()
-      ? constants.ROLE.TRANSCODER
-      : constants.ROLE.DELEGATOR
-  return {
-    role,
-    constants,
-    delegator
+  try {
+    const constants = await protocolService.getLivepeerDefaultConstants()
```

Comment:
Req: I think we need to merge the changes in development in this branch, we can have problemas with the constants right here

Reply:
Done